README.md

```diff
@@ -14,37 +14,113 @@ All bugs, stories, and backlog for this project are managed through [Hyperledger
 in project IS (note that regular Indy tickets are in the INDY project instead...). Also, join
 us on [Hyperledger's Rocket.Chat](https://chat.hyperledger.org/) at #indy-sdk to discuss.
 
-## Building Indy SDK
+
+## How to install
+### Release channels
+There are three stable channels: 
```

Comment:
Still unclear. Just change to:

Indy SDK release process defines the following release channels:

    master - development builds for each push to master branch
    rc - release candidates
    stable - stable releases